admin/sql/CreateTables.sql

```diff
@@ -376,13 +376,20 @@ CREATE TABLE artist_tag_raw
 
 CREATE TABLE artist_credit ( -- replicate
     id                  SERIAL,
+    gid                 UUID NOT NULL,
```

Comment:
This'll have to move to the end of the table definition

Reply:
Fixed at https://github.com/metabrainz/musicbrainz-server/pull/2092/commits/2ad5ae0c4b60d2fd7391bb48188e06a72454a26a#diff-92455c89deea6ff72051d26016d7124841420e89902f3257293b3f142f70ac1cR384